cli/root.go

```diff
@@ -43,6 +52,9 @@ var RootCommand = &cobra.Command{
 			log.Fatal().Msg("invalid target")
 		}
 
+		// Configure custom prefixes if provided
```

Comment:
This comment is not needed, as it doesn't explain anything that is not explained by the method's name already.